package.json

```diff
@@ -263,7 +263,7 @@
     "@datadog/browser-rum": "^4.49.0",
     "@department-of-veterans-affairs/formation": "^12.0.0",
     "@department-of-veterans-affairs/component-library": "^48.0.2",
-    "@department-of-veterans-affairs/css-library": "^0.15.0",
+    "@department-of-veterans-affairs/css-library": "^0.14.0-rc7",
```

Comment:
Once [this PR](https://github.com/department-of-veterans-affairs/component-library/pull/1359) has been merged in component-library, the dependency here will be updated with a non-release candidate version.